project.owasp.yaml

```
@@ -0,0 +1,279 @@
+audience:
+  - builder
+  - defender
+community:
+  - name: proj-dependency-check
+    platform: slack
+    url: https://owasp.slack.com/archives/C2FTJJ26N
+    description: channel for the https://www.owasp.org/index.php/OWASP_Dependency_Check project - For general questions with the tool, check out the mailing
+      list https://groups.google.com/forum/#!forum/dependency-check. To report issues, please submit a GitHub issue.
+  - name: dependency-check
+    platform: slack
+    url: https://owasp.slack.com/archives/C082E5KB4HJ
+leaders:
+  - name: Jeremy Long
+    email: jeremy.long@owasp.org
+    github: jeremylong
+level: 4
+license:
+  - Apache-2.0
+  - EUPL-1.2
+  - GPL-3.0
+  - MIT
```

```suggestion
  - Apache-2.0
  - LGPL-3.0
  - MIT
```

EUPL and GPL have probably been detected from one of the community repos here which are being removed below, so removing.

**Apache 2**: main project, gradle plugin, azuredevops extension, jenkins plugin (if included)
**MIT**: GitHub Action and related repos
**Sonar plugin**: LGPL 3.0 only: https://github.com/dependency-check/dependency-check-sonar-plugin?tab=readme-ov-file#copyright--license There is no "or later" language I can find so should technically be `LGPL-3.0-only` as `LGPL-3.0` is not a valid SPDX identifier, but that's seemingly not in the schema.

---

```suggestion
  - Apache-2.0
  - LGPL-3.0-only
  - MIT
```

---

LGPL-3.0-only isn't in the schema just yet and I think these are validated? I opened an issue at https://github.com/OWASP/nest-schema/issues/593 to discuss in any case

---

I removed "-only" - however, something is still wrong with the yaml as the validation is failing in this section...

---

I don't understand this validator error - looks like code rather than validation errors. How come it's not a standard json-schema validation output? 😅

---

ChatGPT to the rescue! I had it validate the YAML against the json schema ;)

Found the issues and it wasn't with the license. Mostly multi-line descriptions that didn't use the `description: >` syntax in the yaml.